### PR TITLE
Account for moves with <60 Base Power when terastallized

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: sim/battle-actions.ts [hunk 1]
```diff
@@ -1609,6 +1609,17 @@ export class BattleActions {
 			basePower = 0;
 		}
 
+		if (
+			basePower < 60 && source.getTypes(true).includes(move.type) && source.terastallized && move.priority <= 0 &&
```
Comment (urkerab) [inline sim/battle-actions.ts:1613]: `source.getTypes(true)` will always return `source.terastallized` if it's set.

Comment (KrisXV) [reply sim/battle-actions.ts:1613]: I'm unsure what you're saying here

Comment (urkerab) [reply sim/battle-actions.ts:1613]: I don't know what the correct condition is, but `source.getTypes(true).includes(move.type) && source.terastallized` is the same as `move.type === source.terastallized`.

Comment (KrisXV) [reply sim/battle-actions.ts:1613]: Yeah, but the current implementation is used in other places in the sim currently so I wanna keep consistent w/ that

Comment (MathyFurret) [reply sim/battle-actions.ts:1613]: I used the new form in #9138 i like it better

Comment (urkerab) [reply sim/battle-actions.ts:1613]: I couldn't find any other place using `source.getTypes(true)`. I did find one using `source.getTypes(false, true)` but obviously that's completely different.

**Comment by DaWoblefet** [comment]
Thanks Kris!

**Pull request merged**